src/main/groovy/org/shipkit/internal/gradle/git/GitPush.java

```diff
@@ -55,19 +55,6 @@ static void setPushUrl(GitPushTask pushTask, String writeTokenEnvValue, String g
         }
     }
 
-    public static String getWriteToken(ReleaseConfiguration conf, String writeTokenEnvValue) {
-        String token = conf.getLenient().getGitHub().getWriteAuthToken();
```

Comment:
It looks like we may no longer need ```getLenient()``` method on ReleaseConfiguration. Given that it is a public type and hence a liability can you deprecate ```getLenient()``` method? This way it will force us to figure out what to do with it later on :)